doc/index.rst

```diff
@@ -659,6 +659,18 @@ Protocols
 
    .. versionadded:: 4.6.0
 
+.. class:: Reader
+
+    See :py:class:`io.Reader`. Added to the standard library in Python 3.14.
```

Comment:
Please note that these intersphinx links will only start working after Python 3.14 is released.

Reply:
that won't be till the Autumn -- maybe we should hardcode a link to https://docs.python.org/3.14/library/io.html#io.Reader for now, instead of using intersphinx?

Reply:
Or we could point sphinx to 3.14 by default for now.

Reply:
that works for me!